_licenses/epl-2.0.txt

```diff
@@ -0,0 +1,309 @@
+---
+title: Eclipse Public License 2.0
+spdx-id: EPL-2.0
+redirect_from: /licenses/eclipse/
+source: https://www.eclipse.org/org/documents/epl-2.0/EPL-2.0.txt
+hidden: false
+
+description: This commercially-friendly copyleft license provides the ability to commercially license binaries; a modern royalty-free patent license grant; and the ability for linked works to use other licenses, including commercial ones.
+
+how: Create a text file (typically named LICENSE or LICENSE.txt) in the root of your source code and copy the text of the license into the file.
+
+using:
+  - Eclipse SmartHome: https://github.com/eclipse/smarthome/blob/master/LICENSE
```

Comment:
This question applies to all license metadata: why not reference the exact revision, i.e. https://github.com/eclipse/smarthome/blob/36c9e2717e4f89cf60a0d885e7ecc6069b338f48/LICENSE? That way, if the project ever gets relicensed (say, updated to EPL-2.0), the link won't go stale.

Reply:
> … why not reference the exact revision…

+1 to that.  It would be even nicer to reference a tagged release (e.g. once SmartHome cuts 0.9.0).  As consumers are updated to use different licenses, I expect them to be removed from `using`.  But still, where stable links are easy, I think we should be using them instead of floating with `master`.

Reply:
See #555 for a different idea.